Makefile

```
@@ -19,8 +19,6 @@ jupyter_python_version := 3.11
 
 project_name_camel_case := $(shell echo $(project_name) | sed -r 's/(^|-)([a-z])/\U\2/g')
```

This was hard-coded in the IO MR. I'm fine with either as long as we're consistent
```suggestion
project_name_camel_case := Mathematics
```